# area: 5 m^2 max alt: 1.0 m 

/**:
  ros__parameters:
    control_rate: 10.0
    max_velocity: 0.1
    min_velocity: 0.01
    max_acceleration: 0.5
    obstacle_topic: "/obstacle/pose"
    drone_names : 
      - "cf_0"
      - "cf_1"
    wp: 
      a:
        - 1.42
        - 0.18
        - 0.5
        - 0.0
      b:
        - 0.63
        - 1.36
        - 0.5
        - 0.0
      c:
        - -0.81
        - 1.11
        - 0.5
        - 0.0
      d:
        - -1.33
        - -0.27
        - 0.5
        - 0.0
      e:
        - -0.52
        - -1.29
        - 0.5
        - 0.0
      f:
        - 0.94
        - -1.05
        - 0.5
        - 0.0

    charging_stations:
        - 6.0
        - -3.0
        - 0.5
        - 0.0




      # a: 
      #   - 6.55
      #   - 0.95
      #   - 1.4
      #   - 0.0
      # b:
      #   - -0.56
      #   - 4.11
      #   - 1.4
      #   - 0.0
      # c:
      #   - 0.78
      #   - -0.41
      #   - 1.4
      #   - 0.0
      # d:
      #   - -6.17
      #   - 2.0
      #   - 1.4
      #   - 0.0
      # e:
      #   - -8.27
      #   - -4.50
      #   - 1.4
      #   - 0.0
      # f:
      #   - 3.48
      #   - 3.17
      #   - 1.4
      #   - 0.0
